src/elm/Page/Account.elm

```diff
@@ -902,6 +1057,24 @@ subscriptions _ =
 -- INTERNAL
 
 
+iconForPaymentType : PaymentType -> Html Msg
+iconForPaymentType paymentType =
+    case paymentType of
+        Nets Visa ->
+            H.img [ A.src "images/paymentcard-visa.svg" ] []
```

Comment:
Noen grunn til at du gikk for filer i stedet for å fortsette med `Icon`-modulen?

Reply:
Ikke annet enn at jeg testet det og det fungerte bra. Jeg kan godt legge det under `Icon`-modulen? Er de "håndskrevet", eller er det et bibliotek eller lignende som omformer fra svg-filer til elm sin Svg-representasjon?

Reply:
Jeg konverterte de for hånd i hvert fall. Tok ikke lange tiden, så gadd ikke lete etter et verktøy for å gjøre det enklere :)

Reply:
La denne på vent, siden det krevde en god del endringer i RadioGroup/EditSection for å kunne styre om ikoner skal bruke tekstfarge som `fill` eller bruke verdien de som er satt fra før.